drivers/bus/acpi/compbatt/compbatt.c

```diff
@@ -1,9 +1,9 @@
 /*
- * PROJECT:         ReactOS Composite Battery Driver
- * LICENSE:         BSD - See COPYING.ARM in the top level directory
- * FILE:            boot/drivers/bus/acpi/compbatt/compbatt.c
- * PURPOSE:         Main Initialization Code and IRP Handling
- * PROGRAMMERS:     ReactOS Portable Systems Group
+ * PROJECT:     ReactOS Composite Battery Driver
+ * LICENSE:     MIT (https://spdx.org/licenses/MIT)
```

Comment:
COPYING.ARM is actually [BSD-2-Clause-Views](https://spdx.org/licenses/BSD-2-Clause-Views). Is it OK to relicense as MIT?